app/src/js/server/storage.ts

```diff
@@ -13,12 +13,15 @@ export const STORAGE_FOLDERS = [StorageStructure.PROJECT, StorageStructure.USER]
 export abstract class Storage {
   /** the data directory */
   protected _dataDir: string
+  /** whether to use keys as paths directly */
+  protected _useRelativePaths: boolean
```

Comment:
What's the use case of this variable? Is it possible to set `_dataDir` to empty in that case?

Reply:
We could set _dataDir to empty, but there's still the default ".json" extension we want to avoid, since these can be yaml or json files.

Reply:
In this case, add a member method to set the extension is clearer

Reply:
Remove this variable, added a method for setExt